### DISCUSSION TITLE
Can you clone fields from a collection or change a collection name?

### DISCUSSION THREAD

**Discussion by hmirzoyan-chwy** [directus/directus#6514] [Archive]
I want to create a new collection with the same fields as another collection, what are my options?

also.. am I able to change the name of a collection? (not just translate but fully change the name in the DB as well?

**Comment by philleepflorence** [comment]
Hello, hope these help to answer your question.

### Duplicating a collection schema
- Create collection + [Fields Guide: Duplicating a Field](https://docs.directus.io/guides/fields/#duplicating-a-field) via Directus Data Model Settings.
- Copy Collection DB Schema + Export settings from System [Collections](https://docs.directus.io/reference/api/system/collections/) and [Fields](https://docs.directus.io/reference/api/system/fields/) via Direct DB Access, Directus API or [Custom Migration](https://docs.directus.io/guides/migrations/). 

### Changing the name of a collection in the database
- [Collections Guide](https://docs.directus.io/guides/collections/#creating-a-collection) - The collection name, primary key name/type, and system field names can not be modified after the collection is created.
